drivers/sensor/tmp112/tmp112.c

```diff
@@ -188,6 +194,41 @@ static int tmp112_channel_get(const struct device *dev,
 	return 0;
 }
 
+#ifdef CONFIG_DEVICE_POWER_MANAGEMENT
```

Comment:
The code here seems to be against an older version of Zephyr, the Kconfigs and some of the APIs have changed, please update against latest Zephyr tree.

Reply:
I didn't saw that is config has been changed in favor to `CONFIG_PM_DEVICE`. I will also change it in my other open PR